test/integration/init_int_test.go

```diff
@@ -19,54 +23,109 @@ type InitIntegrationTestSuite struct {
 
 func (suite *InitIntegrationTestSuite) TestInit() {
 	suite.OnlyRunForTags(tagsuite.Init, tagsuite.Critical)
-	suite.runInitTest(false, sampleYAMLPython3, "python3")
+	suite.runInitTest(false, "python3", "python3")
 }
 
-func (suite *InitIntegrationTestSuite) TestInit_SkeletonEditor() {
+func (suite *InitIntegrationTestSuite) TestInit_Path() {
 	suite.OnlyRunForTags(tagsuite.Init)
-	suite.runInitTest(false, sampleYAMLEditor, "python3", "--skeleton", "editor")
+	suite.runInitTest(true, "python3", "python3")
 }
 
-func (suite *InitIntegrationTestSuite) TestInit_Path() {
+func (suite *InitIntegrationTestSuite) TestInit_Version() {
 	suite.OnlyRunForTags(tagsuite.Init)
-	suite.runInitTest(true, sampleYAMLPython3, "python3")
+	suite.runInitTest(false, "python3@1.0", "python3")
 }
 
-func (suite *InitIntegrationTestSuite) TestInit_Version() {
+func (suite *InitIntegrationTestSuite) TestInit_DisambiguatePython() {
 	suite.OnlyRunForTags(tagsuite.Init)
-	suite.runInitTest(false, sampleYAMLPython3, "python3@1.0")
+	suite.runInitTest(true, "python", "python3")
+	suite.runInitTest(true, "python@3.10.0", "python3")
+	suite.runInitTest(true, "python@2.7.18", "python2")
 }
 
-func (suite *InitIntegrationTestSuite) runInitTest(addPath bool, config string, language string, args ...string) {
+func (suite *InitIntegrationTestSuite) runInitTest(addPath bool, lang string, expectedConfigLanguage string, args ...string) {
 	ts := e2e.New(suite.T(), false)
 	defer ts.Close()
+	ts.LoginAsPersistentUser()
```

Comment:
Are we removing these newly created projects after the test completes? I _believe_ we are...